public/stylesheets/style.css

```diff
@@ -10,4 +10,24 @@ header {
 a {
   color: #00B7FF; }
 
+* {
+  margin: 0;
+  padding: 0; }
```

Comment:
Is the wildcard style really needed? Wouldn't this introduce unexpected behavior?